lib/widgets.php

```diff
@@ -63,6 +63,14 @@ function gutenberg_block_editor_admin_print_footer_scripts() {
  */
 function gutenberg_block_editor_admin_footer() {
 	if ( gutenberg_is_block_editor() ) {
+		echo implode(
+			"\n",
+			array(
+				'<form method="post">',
+				wp_nonce_field( 'save-sidebar-widgets', '_wpnonce_widgets', false ),
+				'</form>',
+			)
+		);
```

Comment:
What's this for?

Reply:
I added a comment. Basically for the widgets code to be able to call the ajax endpoint we need a nonce.